airbyte-integrations/connectors/destination-kafka/src/main/resources/spec.json

```diff
@@ -0,0 +1,223 @@
+        "type": "string",
+        "examples": ["sample.topic", "{namespace}.{stream}.sample"],
+        "order": 1
+      },
+      "test_topic": {
+        "title": "Test topic",
+        "description": "Topic to test if Airbyte can produce messages.",
+        "type": "string",
+        "examples": ["test.topic"],
+        "order": 2
+      },
+      "sync_producer": {
+        "title": "Sync producer",
+        "description": "Wait synchronously until the record has been sent to Kafka.",
+        "type": "boolean",
+        "default": false,
+        "order": 3
+      },
+      "security_protocol": {
+        "title": "Security protocol",
+        "description": "Protocol used to communicate with brokers.",
+        "type": "string",
+        "default": "PLAINTEXT",
+        "enum": ["PLAINTEXT", "SASL_PLAINTEXT", "SSL", "SASL_SSL"],
+        "order": 4
+      },
+      "sasl_jaas_config": {
+        "title": "SASL JAAS config",
+        "description": "JAAS login context parameters for SASL connections in the format used by JAAS configuration files.",
+        "type": "string",
+        "default": "",
+        "airbyte_secret": true,
+        "order": 5
+      },
+      "sasl_mechanism": {
+        "title": "SASL mechanism",
+        "description": "SASL mechanism used for client connections. This may be any mechanism for which a security provider is available.",
+        "type": "string",
+        "default": "PLAIN",
+        "enum": ["PLAIN", "GSSAPI"],
+        "order": 6
+      },
+      "client_id": {
+        "title": "Client ID",
+        "description": "An id string to pass to the server when making requests. The purpose of this is to be able to track the source of requests beyond just ip/port by allowing a logical application name to be included in server-side request logging.",
+        "type": "string",
+        "examples": ["airbyte-producer"],
+        "order": 7
+      },
+      "acks": {
+        "title": "ACKs",
+        "description": "The number of acknowledgments the producer requires the leader to have received before considering a request complete. This controls the  durability of records that are sent.",
+        "type": "string",
+        "default": "1",
+        "enum": ["0", "1", "all"],
+        "order": 8
+      },
+      "transactional_id": {
+        "title": "Transactional ID",
+        "description": "The transactional ID to use for transactional delivery. This enables reliability semantics which span multiple producer sessions since it allows the client to guarantee that transactions using the same transactional ID have been completed prior to starting any new transactions. If no transactional ID is provided, then the producer is limited to idempotent delivery.",
+        "type": "string",
+        "default": "",
+        "examples": ["txn-id"],
+        "order": 9
+      },
+      "transaction_timeout_ms": {
```

Comment:
does the user need to specify this or can we use a default?

Reply:
I removed this property to avoid transactions so it's not needed anymore